src/style.js

```diff
@@ -141,6 +147,17 @@ export function applyDirectStyles(selection, mark) {
   applyAttr(selection, "opacity", mark.opacity);
 }
 
+function applyHref(selection, href, target) {
+  selection.each(function(i) {
+    if (href[i] != null) {
+      const a = document.createElementNS(namespaces.svg, "a");
+      a.setAttributeNS(namespaces.xlink, "href", href[i]);
+      if (target != null) a.setAttribute("target", target);
+      this.parentNode.insertBefore(a, this).appendChild(this);
```

Comment:
makes we wish we could do selection.wrap("svg:a") (related: https://github.com/d3/d3-selection/pull/296)

Reply:
selection.wrap: https://github.com/d3/d3-selection/pull/297